Packages/MIES/MIES_ProgrammaticGUIControl.ipf

```diff
@@ -235,7 +235,10 @@ Function PGC_SetAndActivateControl(win, control, [val, str, switchTab])
 
 	// call only once
 	ControlInfo/W=$win $control
-	ASSERT(V_flag != 0, "Non-existing control or window")
+	if(!V_flag)
+		ASSERT(WindowExists(win), "The panel " + win + " does not exist.")
```

Comment:
@t-b it'll be a bit tedious for the user if the win and control don't exist but I can't think of a better way to do it.

Reply:
@timjarsky I can fix that ;)

Reply:
@timjarsky Sorry I amend that statement. This is not fixable as if the windows does not exist I can't check for the control.